### DISCUSSION TITLE
what is the priority of removing openjdk dependencies, is there a specific time ?

### DISCUSSION THREAD

**Discussion by Bidaya0** [batect/batect#1205] [Ideas]
Because I'm using batect on a django project, it's based on python and doesn't need openjdk.
[https://github.com/HXSecurity/DongTai-webapi](https://github.com/HXSecurity/DongTai-webapi)
It would be weird when someone else collaborates on a python project but needs an openjdk for development.
I had the same problem, when advising our frontend developers to use batect.
If there is a problem with the kotlin native, is it possible to consider using openjdk in docker and use dockerindocker like [this](https://zhuanlan.zhihu.com/p/336977892)

**Comment by charleskorn** [comment]
Hi @Bidaya0, this is on my roadmap, but I have no immediate plans to implement it. My current focus is on replacing the Docker client used by Batect, and after that I'm planning to tackle #990 and #991. 

Once those are out of the way, my plan is to look into Kotlin/Native more seriously, but I have no timeframe to share at the moment.
